Makefile

```diff
@@ -297,22 +309,35 @@ release/bundle-image:
 	bazel run --stamp --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 \
 		//:push_operator_bundle_image
 
-
-OLM_REPO ?=
-OLM_BUNDLE_REPO ?= cockroachdb-operator-index
-OLM_PACKAGE_NAME ?= cockroachdb-certified
-TAG ?= $(APP_VERSION)
+# This target:
+# 1. Updates CRD and CSV files 
+# 2. Pushes the operator image to a registry
+# 3. Builds the OpenShift bundle
+# 4. Pushes the OpenShift bundle to a registry
+# 5. Run opm to create and push the OpenShift index container to a registry
+# 6. Removes the newly created OpenShift files so that it can run again. 
 #
-# dev opm index build for quay repo
+# The following env variables are used for the above process.
 #
-.PHONY: dev/opm-build-index
-dev/opm-build-index:
-	RH_BUNDLE_REGISTRY=$(RH_BUNDLE_REGISTRY) \
-	RH_BUNDLE_IMAGE_REPOSITORY=$(RH_BUNDLE_IMAGE_REPOSITORY) \
-	RH_BUNDLE_VERSION=$(RH_BUNDLE_VERSION) \
-	RH_DEPLOY_PATH=$(RH_DEPLOY_FULL_PATH) \
+# APP_VERSION
+# VERSION
+# RH_BUNDLE_VERSION
+# RH_OPERATOR_IMAGE
+# DOCKER_REGISTRY
```

Comment:
Where are these set?

Reply:
We have values in the Makefile that are defaults. Otherwise you can override them on the command line. You can take a look at my script that I am using to run the test.